drive/files.py

```diff
@@ -294,22 +305,22 @@ def to_dict(self):
 
         return d
 
-    def __str__(self):
-        return self.name
+    def __str__(self) -> str:
+        return self.name or ''
```

Comment:
I think this is the only spot where I really changed behavior (other than returning false one place where `self._client` is missing). Not sure what you’d like to see here when `name` is missing, but the previously implementation made MyPy unhappy since this should always be returning a string.